edx_solutions_api_integration/imports/views.py

```diff
@@ -193,7 +193,17 @@ def _enroll_user(self, data, errors):
         try:
             cohort = add_cohort(course_key, CourseUserGroup.default_cohort_name, CourseCohort.RANDOM)
```

Comment:
Couldn't it just be `atomic` instead of retrying it later?

Reply:
I'm not sure I understood. How would that solve the problem of the theory that the read replica is out of date for a bit?

Reply:
Ah, right, in this case it would not resolve it.